### PR TITLE
Display MPS benchmark results in eager mode

### PR DESCRIPTION
**Pull request by huydhn**
This goes together with https://github.com/pytorch/pytorch/pull/142034.  MPS benchmark is running in eager mode at the moment.

### CODE REVIEW

**Review by ZainRizvi — COMMENTED** [review]
which page is this expected to change? Mind adding before/after screen shots?

**Review by ZainRizvi — APPROVED** [review]
Is the MPS view expected to only have a single datapoint for that week long time window?

**Comment by huydhn** [comment]
> Is the MPS view expected to only have a single datapoint for that week long time window?

Oh, the data only shows up on the dashboard correctly after I land https://github.com/pytorch/pytorch/pull/142034 yesterday, and the job is scheduled daily. So, there is only one data point atm.

File: torchci/components/benchmark/compilers/common.tsx [hunk 1]
```diff
@@ -38,8 +39,9 @@ export const DISPLAY_NAMES_TO_COMPILER_NAMES: { [k: string]: string } = {
   cudagraphs_freezing: "inductor_with_cudagraphs_freezing",
   cudagraphs_low_precision: "inductor_cudagraphs_low_precision",
   export: "inductor_export",
+  eager: "inductor_eager",
 };
-export const BLOCKLIST_COMPILERS = ["aot_eager", "eager"];
```
Comment (huydhn) [inline torchci/components/benchmark/compilers/common.tsx:42]: TorchInductor uses to blocklist eager

**Pull request merged**